### PR TITLE
3554 ms is ideal number

### PR DESCRIPTION
**Pull request by rbtemp**
That's THE value for blinking delay.

### CODE REVIEW

File: blink.ino [hunk 1]
```diff
@@ -19,7 +19,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   if (delay >= 2000) {
```
Comment (RichardBenes) [inline blink.ino:21]: Here it does not look correct. The delay should be static.

File: blink.ino [hunk 2]
```diff
@@ -8,7 +8,7 @@
 // Pin 13 has an LED connected on most Arduino boards.
 // give it a name:
 int led = 13;
-unsigned int delay = 1000;
+unsigned int delay = 3554;
```
Comment (RichardBenes) [inline blink.ino:11]: The delay value should be inlined. Magic numbers are what we want.

File: blink.ino [hunk 3]
```diff
@@ -19,7 +19,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   if (delay >= 2000) {
-  	delay = 1000;
+  	delay = 3554;
   }  
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
   delay(delay);               // wait for a second
```
Comment (RichardBenes) [inline blink.ino:25]: Here the magic val should be.

**Review by RichardBenes — CHANGES_REQUESTED** [review]
It was a nice try. Let's finish it!

**Comment by rbtemp** [comment]
I did just about everything. Is it correct now?

File: blink.ino [hunk 4]
```diff
@@ -18,12 +17,8 @@ void setup() {
 
 // the loop routine runs over and over again forever:
 void loop() {
-  if (delay >= 2000) {
-  	delay = 1000;
-  }  
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(delay);               // wait for a second
+  delay(3554);               // wait for a second
```
Comment (RichardBenes) [inline blink.ino:21]: Yes, this is sooo magic.

**Review by RichardBenes — APPROVED** [review]

**Pull request merged**